lib/css/components/_stacks-toggle-switches.less

```diff
@@ -10,76 +10,142 @@
 //  $   BASE STYLE
 //  ----------------------------------------------------------------------------
 .s-toggle-switch {
-    display: flex;
-    align-items: stretch;
-
-    //  First hide the radio buttons
-    input[type="radio"] {
-        position: absolute;
-        left: -999em;
+    input[type="checkbox"] {
+        // "Hide" it by moving it out of sight
+        // This allows users to still "tab" to it, interact via keyboard, etc
         opacity: 0;
-        margin: 0;
-    }
+        position: absolute;
+        left: -99999999px;
```

Comment:
I don't like line this because is provides a weird and inconsistent visual artifact while navigating with VoiceOver.

- Chrome has a visual indicator that sits at the left edge of the browser.
- Safari has a visual indicator extending from the left edge of the browser to the left edge of the **screen**.
- FireFox has a visual indicator surrounding the whole browser window.

![Screen Shot 2021-02-19 at 11 14 03 AM](https://user-images.githubusercontent.com/662837/108546164-f88b9680-72a5-11eb-8117-fa6dec43903e.png)

 An alternative would be:

```less
.s-toggle-switch {
    position: relative;

    input[type="checkbox"] {
        position: absolute;
        left: 0;
        top: 0;
        width: 44px;
        height: 24px;
    }
}
```

This positions the invisible checkbox exactly under the visual checkbox, giving it the following visual indicators:

- Around the toggle + label in Safari
- Around the toggle in Chrome/Firefox

![Screen Shot 2021-02-19 at 11 36 19 AM](https://user-images.githubusercontent.com/662837/108546798-dc3c2980-72a6-11eb-8528-294811b51912.png)
![Screen Shot 2021-02-19 at 11 35 44 AM](https://user-images.githubusercontent.com/662837/108546821-e3633780-72a6-11eb-90ee-652b081679a3.png)

Placing the input directly under the graphic also has a practical benefit...